homework/HackYourTemprature/package.json

```
@@ -0,0 +1,22 @@
+{
+  "name": "hackyourtemprature",
```

```suggestion
  "name": "hackyourtemperature",
```

English speakers are lazy and don't say it but there is an `e` hiding in there :laughing:

---

Even the spelling director fall in this laziness 😂